browser/components/pages/RequestReviewPage.js

```diff
@@ -3,13 +3,16 @@ import Button from '../atoms/Button'
 import Layout from '../molecules/Layout'
 import InspectObject from '../utils/InspectObject'
 import createPrrr from '../../actions/createPrrr'
+import './RequestReviewPage.sass'
 
 export default class RequestReviewPage extends Component {
   render(){
     const { session } = this.props
-    return <Layout className="HomePage" session={session}>
+    return <Layout className="RequestPage" session={session}>
+      <div className="RequestBox" >
```

Comment:
I'd like to keep all classnames scoped under `RequestReviewPage` so lets make this `RequestReviewPage-RequestBox`